fs/fs.go

```diff
@@ -47,26 +45,6 @@ const (
 	LabelCrioImages   = "crio-images"
 )
 
-// The maximum number of `du` and `find` tasks that can be running at once.
-const maxConcurrentOps = 20
```

Comment:
I am pretty concerned about removing the limit on the number of concurrent calls...  Especially now that the filesystem walk is no longer being run with `nice` and `ionice`, this has the potential to starve everything else on the node of cpu or blkio.

Reply:
I think the pure go way to implement this, for what it's worth, would be to do one of the following:

1. Use something like [docker/pkg/reexec](https://github.com/moby/moby/tree/v17.05.0-ce/pkg/reexec) to fork the kubelet into a mode where it just does stats and return the result to the parent kubelet (likely over stdout).
     This has the benefit that we could also eventually move it into the pod's cgroup, or at least it's own non-kubelet cgroup, and do memory/cpu accounting too for it.
    This has the downside that it would require some extra setup code in the kubelet to correctly deal with re-exec stuff, and last time I checked kubernetes had no existing infrastructure for this.

2. Use [runtime.LockOSThread](https://golang.org/pkg/runtime/#LockOSThread) and set nice on the one thread doing this stuff.. I think as of [go1.10](https://github.com/golang/go/issues/20676) it's possible to do that safely, though historically it has been fraught with peril.

Reply:
With CRI-based runtimes, the disk metrics come from the runtime, rather than cAdvisor, so I don't think (1) is really worth it.